bin/gwd/request.ml

```diff
@@ -72,6 +72,7 @@ let make_senv conf base =
 let family_m conf base =
   let open RequestHandler in
   let handler = H.handler in
+<<<<<<< HEAD
```

Comment:
You have a conflict relicat here.

Also, please do not indent the code. I know it would be wrongly indented, but its better in order to review the PR. We can fix the indentation later, when the change will be merged.

Reply:
done